include/rcpputils/filesystem_helper.hpp

```diff
@@ -271,7 +271,7 @@ RCPPUTILS_PUBLIC path temp_directory_path();
  * This guarantees that there will be no existing files in the returned directory.
  *
  * \param base_name User-specified portion of the created directory
- * \param parent The parent path of the directory that will be created
+ * \param parent_path The parent path of the directory that will be created
```

Comment:
Can I suggest that we change all of the parameter lines in this file to be `\param[in]`, `\param[out]`, or `\param[inout]`?  That will be a bigger patch, but will be more consistent with what we use elsewhere in this package.

Reply:
Yes, that makes sense. Addressed in 22eeba1. `rcpputils/assert.hpp` received the same treatment for consistency.